### PR TITLE
Add research article with aside element

### PR DESCRIPTION
**Pull request by viktoria-mahmud**

### CODE REVIEW

File: assets/sass/patterns/molecules/article-section.scss [hunk 1]
```diff
@@ -373,6 +373,60 @@ $arrow-size: 5;
   }
 }
 
+.section-wrapper {
+  @media only all and (min-width: #{get-rem-from-px($bkpt-site--small)}em) {
+    display: grid;
```
Comment (discodavey) [inline assets/sass/patterns/molecules/article-section.scss:378]: As seeing as we are starting to use this in a few places we could make a mixin with lines 378,379 and 380

Comment (viktoria-mahmud) [reply assets/sass/patterns/molecules/article-section.scss]: Good idea, thanks, Dave! The mixin grid is created in order to reuse the logic for grid display. The files info-bars.scss and content-header-journal.scss are not related to this PR but they can use the mixin too.

File: assets/sass/patterns/molecules/article-section.scss [hunk 2]
```diff
@@ -373,6 +373,62 @@ $arrow-size: 5;
   }
 }
 
+.section-wrapper {
```
Comment (Cosmin-Daniel) [inline assets/sass/patterns/molecules/article-section.scss:376]: A max-width and margin: 0 auto; may be needed here, as in the .content-header.

Comment (viktoria-mahmud) [reply assets/sass/patterns/molecules/article-section.scss:376]: The class "section-wrapper" is updated with these styles, the content header will be handled in another task, now I am reusing an existing variant just for visual presentation

**Review by Cosmin-Daniel — APPROVED** [review]
Looks good to me.

File: source/_patterns/04-pages/article--research-aside.mustache [hunk 3]
```diff
@@ -0,0 +1,91 @@
+<div class="global-wrapper" data-item-type="research-article">
```
Comment (viktoria-mahmud) [inline source/_patterns/04-pages/article--research-aside.mustache:1]: After a call with Nathan, I created a new page for aricle research pages with aside element, the original page for article research was reverted

**Review by discodavey — COMMENTED** [review]
This is just a question so I understand. Why is the grid in the article section in molecules as the content journal grid is in organisms?

**Comment by viktoria-mahmud** [comment]
> This is just a question so I understand. Why is the grid in the article section in molecules as the content journal grid is in organisms?

Good point, I created a new acticle page so it is better to have new file with the styles too. You can find them in aside-section-wrapper.scss under organisms folder.

**Pull request closed without merging**